com.github.miguelmota.Cointop.json

```diff
@@ -0,0 +1,68 @@
+{
+  "app-id": "com.github.miguelmota.Cointop",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.golang"
+  ],
+  "command": "cointop",
+  "cleanup": [
+    "/go",
+    "/bin/scripts"
+  ],
+  "modules": [
+    {
+      "name": "scripts",
+      "sources": [
+        {
+          "type": "script",
+          "commands": [
+            ". /usr/lib/sdk/golang/enable.sh;  export GOPATH=/app/go"
+            ],
+            "dest-filename": "my_enable.sh"
+        },
+        {
+          "type": "script",
+          "commands": [
+            ". /app/bin/scripts/my_enable.sh; cd /app/go/src/$1; go build -o x"
+            ],
+            "dest-filename": "build.sh"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/go/{src,pkg,bin} # make go workspace",
+        "install -d /app/bin/scripts",
+        "cp *.sh /app/bin/scripts/"
+      ]
+    },
+    {
+      "name": "hello",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cp -rpv go/* /app/go/",
+        "/app/bin/scripts/build.sh github.com/miguelmota/cointop",
+        "install -D /app/go/src/github.com/miguelmota/cointop/x /app/bin/cointop",
+        "mkdir -p /app/share/icons/hicolor/64x64/apps",
+        "mkdir -p /app/share/icons/hicolor/128x128/apps",
+        "mkdir -p /app/share/applications",
+        "cp /app/go/src/github.com/miguelmota/cointop/assets/icon_64x64.png /app/share/icons/hicolor/64x64/apps/com.github.miguelmota.Cointop.png",
+        "cp /app/go/src/github.com/miguelmota/cointop/assets/icon_128x128.png /app/share/icons/hicolor/128x128/apps/com.github.miguelmota.Cointop.png",
+        "cp /app/go/src/github.com/miguelmota/cointop/flathub/com.github.miguelmota.Cointop.desktop /app/share/applications/com.github.miguelmota.Cointop.desktop"
+      ],
```

Comment:
Forgetting to install the appdata file into `/app/share/metainfo`

Reply:
added